app/controllers/data_tables_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class DataTablesController < ApplicationController
+  include BreadcrumbBuilder
+
+  # def index
+  #   Show a list of all data table records sorted by creation date, so that
+  #   users will be able to access historical data
+  # end
+
+  def show
+    # Add id parsing when we'll need to let the user download data tables other
+    # than the latest
+    @data_table = DfEDataTable.last
```

Comment:
This will not reliably return the latest `DfEDataTable`, because UUIDs aren't sequential.

Reply:
I think...

Reply:
Just to be on the safe side, I added `order(created_at: asc)` as a `default_scope` in the model, this way it should always sort with the oldest first and the more recent last unless explicitly `unscoped`.
Do you confirm it's the expected behaviour?

Reply:
Indeed, from https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-last: 

>  If no order is defined it will order by primary key.